packages/docs/docs/communications/organizing-communications/organizing-communications.md

```diff
@@ -0,0 +1,167 @@
+# Organizing Communications
+
+## Background
+
+Communication in a patient portal can include many scenarios – patient to physician, physician to physician, device to physician, and more. This results in a large volume of communications within a portal, so ensuring that they are well organized is vital. For example, an organization may have staff on call to respond to patient messages. Without proper organization, messages may not reach the correct staff member.
```

Comment:
Patient portal is perhaps misleading here as these are not necessarily in a patient portal itself.  Perhaps say something like "Communications in a healthcare context".  Similar with second sentence.

Reply:
Also note here that communications are modeled as Communication resources and link to to the Communication resource at `/docs/api/fhir/resources/communication`